tests/end_to_end/result.coffee

```diff
@@ -194,3 +192,39 @@ describe 'Result Routes', ->
               value: Joi.string()
               count: Joi.number()
           )
+
+  describe 'supports custom assignments view counter', ->
+    before ->
+      Conversion.remove().exec()
+
+    it 'counts', ->
+      from = new Date()
+      from.setDate(from.getDate() - 7)
+      to = new Date()
+      to.setDate(to.getDate() + 1)
+
+      queryParams = "event=engaged_gameplay&param=login_button&from=#{from}" +
+                    "&to=#{to}&viewCounter=assigned"
+
+      yesterday = new Date()
+      yesterday.setDate(yesterday.getDate() - 1)
+      flare
+        .post '/experiments', {userId: 123}
+        .expect 200
+        .post '/experiments', {userId: 129}
+        .expect 200
+        .post '/experiments', {userId: 132}
+        .expect 200
```

Comment:
This doesn't seem to test that a user forced into an experiment doesn't count as an assigned 'view'